dist/linux/common/org.cryptomator.Cryptomator.tray-unlocked.svg

```diff
@@ -1,12 +1,11 @@
-		<path d="m39 28h-10v-4a3.13 3.13 0 0 1 3-3 3.087 3.087 0 0 1 3 3v1a1.034 1.034 0 0 0 1 1h1a1.034 1.034 0 0 0 1-1v-1a6 6 0 0 0 -12 0v4h-1a2.073 2.073 0 0 0 -2 2v6a2.073 2.073 0 0 0 2 2h14a2.073 2.073 0 0 0 2-2v-6a2.073 2.073 0 0 0 -2-2zm-5.391 5.94a1.609 1.609 0 0 1 -3.217 0v-1.876a1.609 1.609 0 0 1 3.217 0z"/>
-	</g>
+	<defs id="defs3051">
+      <style type="text/css" id="current-color-scheme">
+        .ColorScheme-Text {
+          color:#232629;
+        }
+        </style>
+    </defs>
+		<path style="fill:currentColor;fill-opacity:1;stroke:none" class="ColorScheme-Text" d="m15.591 35.824c-.019.009-.936.775-1.458 1.208a.418.418 0 0 1 -.627-.111 9.322 9.322 0 0 1 -.3-5.974 15.843 15.843 0 0 0 2.894 2.043c.051 1.03-.161 2.644-.509 2.834zm6.409-6.824h-2l.5-5a2 2 0 1 1 1 0zm-14.544-3.241.744-1.366a1.579 1.579 0 0 0 -.019-1.557l.653-1.2c.2.014-.03-.113.165-.14.051-.217-.051-.336 0-.5a3.269 3.269 0 0 0 0-1.5 7.151 7.151 0 0 1 0-3 2.366 2.366 0 0 0 -2.378 1.448 2.409 2.409 0 0 0 .229 2.661l-.7 1.278a1.779 1.779 0 0 0 -1.317.891l-.741 1.372a1.577 1.577 0 0 0 -.019 1.487 3.028 3.028 0 0 0 -2.746 1.525 2.648 2.648 0 0 0 .044 2.631.748.748 0 0 0 .981.266.656.656 0 0 0 .284-.92 1.37 1.37 0 0 1 -.023-1.361 1.6 1.6 0 0 1 2.079-.63 1.408 1.408 0 0 1 .672 1.95 1.546 1.546 0 0 1 -1.2.78.688.688 0 0 0 -.636.749.707.707 0 0 0 .717.6.789.789 0 0 0 .082 0 2.989 2.989 0 0 0 2.322-1.513 2.669 2.669 0 0 0 -.377-3.084 1.767 1.767 0 0 0 1.184-.867zm13.544-10.759a13.013 13.013 0 0 1 5-1 21.6 21.6 0 0 1 4.5.5 9.312 9.312 0 0 0 -9.5-8.5c-5.794 0-9.176 4-9.5 8.5a21.858 21.858 0 0 1 4.5-.5 12.819 12.819 0 0 1 5 1zm3.5-5c1.209 0 2.5.866 2.5 2h-5c0-1.134 1.291-2 2.5-2zm-7 0c1.209 0 2.5.866 2.5 2h-5c0-1.134 1.291-2 2.5-2zm14.473 6a8.067 8.067 0 0 0 -8.08 8v2.141a3.891 3.891 0 0 0 -2.893 3.734v5.125a23.166 23.166 0 0 1 -4.174-1.623 7.857 7.857 0 0 1 -.027.878 3.263 3.263 0 0 1 -.729 2.074l-1.794 1.483a.379.379 0 0 1 -.276.188h-4c-1.324 0-2.346-1.336-2.653-3.343a7.058 7.058 0 0 1 .234-3.18 3.477 3.477 0 0 1 1.636-2.157 1.868 1.868 0 0 1 .783-.32h1.5a8.035 8.035 0 0 1 -1.5-5 11.1 11.1 0 0 1 .5-3 2.519 2.519 0 0 0 0-1.5 13.272 13.272 0 0 1 -.5-3.5c6.687-1.936 11 0 11 0s4.319-1.955 11 0"/>
+		<path style="fill:currentColor;fill-opacity:1;stroke:none" class="ColorScheme-Text" d="m39 28h-10v-4a3.13 3.13 0 0 1 3-3 3.087 3.087 0 0 1 3 3v1a1.034 1.034 0 0 0 1 1h1a1.034 1.034 0 0 0 1-1v-1a6 6 0 0 0 -12 0v4h-1a2.073 2.073 0 0 0 -2 2v6a2.073 2.073 0 0 0 2 2h14a2.073 2.073 0 0 0 2-2v-6a2.073 2.073 0 0 0 -2-2zm-5.391 5.94a1.609 1.609 0 0 1 -3.217 0v-1.876a1.609 1.609 0 0 1 3.217 0z"/>
```

Comment:
Is moving the style to each node part of the fix? I would rather deduplicate it, applying it to the group element.

Reply:
Same argument here like above: I used a KDE Breeze icon theme symbolic icon as a blueprint. The KDE symbolic icons do work correctly when it comes to changing their color on theme changes. See this example with repeated style attributes: https://invent.kde.org/frameworks/breeze-icons/-/blob/master/icons/actions/16/path-reverse.svg?ref_type=heads&plain=1

Reply:
This might not be optimal from a technical point of view, but KDE lacks a clear documentation on how to create symbolic icons, that change their color on theme changes. And I could not find hints on that on the web, e.g. for other DEs.

Reply:
I would like to understand what causes the (mis)behaviour. If I find some time, I'll do a little diff bisecting to see which part of the change is actually responsible for the fix

Reply:
Let me do that for you, as I have an existing testing environment, that speeds up bisecting.

Here are the results:
1. As noted before, the `style` attributes within the `path` tags cause the issue and need to be removed
2. The `defs` tag is required, today I was able to find [some documentation](https://develop.kde.org/docs/plasma/theme/theme-colors/) on that
3. we can use the `g` tag to group the `path` tags
4. It's not sufficient to have `class="ColorScheme-Text"`, instead `class="ColorScheme-Text" fill="currentColor"` is needed

Reply:
That is useful! Thank you!

If you could restore the original indentation, the diff should show just the actual changes :)

Reply:
What would be extra fancy: Using `ColorScheme-Highlight` for the lock in the "unlocked" variant 😄

However, I am not sure if this is just a KDE thing and will still work with other DEs...

Reply:
> What would be extra fancy: Using `ColorScheme-Highlight` for the lock in the "unlocked" variant 😄
> 
> 
> 
> However, I am not sure if this is just a KDE thing and will still work with other DEs...

I'll check that.

Reply:
Resolved by https://github.com/cryptomator/cryptomator/pull/3785